code/modules/food_and_drink/snacks.dm

```diff
@@ -2892,3 +2892,49 @@ ABSTRACT_TYPE(/obj/item/reagent_containers/food/snacks/soup)
 	initial_volume = 20
 	initial_reagents = "currypowder"
 	food_effects = list("food_hp_up","food_refreshed","food_warm")
+
+
+/obj/item/reagent_containers/food/snacks/cheesewheel
+	name = "cheese wheel"
+	desc = "A giant wheel of cheese. It seems a slice is already missing."
+	icon = 'icons/obj/foodNdrink/food_meals.dmi'
+	icon_state = "cheesewheel"
+	inhand_image_icon = 'icons/mob/inhand/hand_food.dmi'
+	throwforce = 1
```

Comment:
This is also default item throwforce
also it's a damn cheese wheel give it some HEFT